src/main/java/org/mskcc/cbio/oncokb/querydomain/UserTokenUsageWithInfo.java

```diff
@@ -0,0 +1,14 @@
+package org.mskcc.cbio.oncokb.querydomain;
+
+import org.mskcc.cbio.oncokb.domain.Token;
+
+/**
+ * Created by Yifu Yao on 2021-1-14
+ */
+
+public interface UserTokenUsageWithInfo {
```

Comment:
An interface without actual implementation, this is new to me. Could you give me a reference how this works?

Reply:
I created it with reference to `UserTokenUsage`, which is also an interface without actual implementation.